README.md

```diff
@@ -436,7 +436,13 @@ One attempt is to allow several privileges until the setup works. Than reduce pr
 **2.** Init and DBus
  - A few applications need a [DBus](#dbus) user daemon. Install `dbus` in image and try option `--dbus`.
  - A few applications need systemd and/or a running [DBus](#dbus) system daemon. Install `systemd` in image and try option `--init=systemd`.
-   
+
+**3.** Docker Image & OS Architecture Mismatch
+- The Docker image may not have been built for the architecture you are currently running on (ie. Image is built for amd64 but you are running on arm). 
+  - You can check the architecture of an image with `docker inspect --format {{.Architecture}} IMAGENAME`
+  - With a mismatch the container will quit unexpectedely & `x11docker` may emit the error `x11docker ERROR: dockerrc(): Did not receive PID of PID1 in container.`
+- Use or build a Docker Image that supports your target architecture.
```

Comment:
```suggestion
  - Use or build a Docker Image that supports your target architecture.
```

Reply:
Fixed Indentation in d2e7c0dfadf1e0de9d0af216a16b3b18cfcb97cd